declaration.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.scss" {
```

Comment:
Do we need this file? Are we really going to use scss?

(I recommend not)

Reply:
Probably not. This is leftover from before I added nested flag to PostCSS :)

Reply:
I think we should avoid scss if we can since the native dependency is a pain

Reply:
I removed it!